### DISCUSSION TITLE
Access players in Home Assistant script/automations MA Beta 2.0.0b73

### DISCUSSION THREAD

**Discussion by NWSS** [music-assistant/support#1531] [Q&A] [answered]
First off, I'm really enjoying MA! Thanks for all the hard work.

Question: I'd like to play music started with automations. I have a zigbee remote control which I'd like to use to start some music. But I can't seem to find the players/devices in Home Assistant. I'm using players that are all airplay accessed. Is there a reason they don't show up in HA? In MA they work perfectly.

**Comment by OzGav** [comment, marked as answer]
You have to add the MA integration via HACS. This will create players in HA which will likely duplicate other players you already have but you need to target these new players.

**Comment by NWSS** [comment]
Perfect! Thanks!

**Comment by NWSS** [comment]
Can I ask a follow up question. Can I use commands like sync etc. in Home Assistant in automations? Join doesn't do it, but in the UI of MA the sync function does work.

**Reply by OzGav** [reply]
OK I think you are talking about linking players to allow synced playback. No you can't do that at the moment. Not sure of your use case and I don't use it but if you create a universal group you might be able to mute and unmute (or power on / off)  indivdual players which would kind of mimic creating different groups?

**Comment by Dustinhoefer** [comment]
I am having the same issue, the players created by MA do not offer any functionality as an action for an automation :/

Am I missing something? Is this not yet supported?

**Reply by OzGav** [reply]
What do you mean by “do not offer any functionality as an action”? The MA players are used as the target entity. You can then play a playlist or album (for example) from MA.
